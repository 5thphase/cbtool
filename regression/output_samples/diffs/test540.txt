 status: Checking ssh accessibility on vm_43 (ssh klabuser@xxx.xxx.xxx.xxx )...
 status: Bootstrapping vm_43 (creating file cb_os_paramaters.txt in "klabuser" user's home dir on xxx.xxx.xxx.xxx )...
 status: Sending a copy of the code tree to vm_43 (xxx.xxx.xxx.xxx )...
 status: Performing generic application instance post_boot configuration on all VMs belonging to ai_1...
 status: Running application-specific "resize" configuration on all VMs belonging to ai_1...
 status: QEMU Scraper will NOT be automatically started during the deployment of ai_1...
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1456183854 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1456183861 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1456183873 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1456183917 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1456183935 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1456183941 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|name |role |size |cloud_ip |host_name |vmc_pool |netname |ai |aidrs |uuid
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |ai_XX |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |ai_XX |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |ai_XX |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |ai_XX |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |ai_XX |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |ai_XX |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |ai_XX |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |ai_XX |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |ai_XX |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |ai_XX |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |ai_XX |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |ai_XX |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |ai_XX |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |ai_XX |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |ai_XX |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |ai_XX |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |ai_XX |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|vm_XX |type |sizeXX |xxx.xxx.xxx.xxx |simhostX |POOL |private |ai_XX |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ssh_key_name |cbtool_rsa 
|Redis Version |2.8.6 
|HOSTs |60 
|VMCs ISSUED |16 
|HOSTs ISSUED |0 
|VMs ISSUED |43 
|VMs SLA PROVISIONING VIOLATED |0 
|VMs SLA RUNTIME VIOLATED |0 
|VMs APPLICATION ERRORS |7 
|AIs ISSUED |6 
|AIDRSs ISSUED |0 
|VMCRSs ISSUED |0 
|FIRSs ISSUED |0 
|EXPERIMENT COUNTER |279 
