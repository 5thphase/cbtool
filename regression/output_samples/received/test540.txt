 status: Waiting for vm_43 (cloud-assigned uuid C54E2C1A-1C5F-583B-B275-18F33D5A5794) to start...
 status: Trying to establish network connectivity to vm_43 (cloud-assigned uuid C54E2C1A-1C5F-583B-B275-18F33D5A5794), on IP address 71.32.25.219...
 status: Bypassing the sending of a copy of the code tree to vm_43 (71.32.25.219)...
 status: Bypassing application-specific "setup" operations
AI object 6443665D-20F9-5426-BE46-4264C8DDF367 (named "ai_1") successfully resized on this experiment.
The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_1      |daytrader      |1_x_client_daytrader->2_x_was->1_x_db2            |19.171.57.58    |1392410635     |none                                    |6443665D-20F9-5426-BE46-4264C8DDF367
|ai_2      |hadoop         |1_x_hadoopmaster->3_x_hadoopslave                 |93.29.116.120   |1392410640     |none                                    |721E9CE0-12BC-5693-90B6-09CBAF69B856
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |100.50.216.252  |1392410653     |none                                    |33568B9B-066D-5F6D-9057-69C07C64B54D
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |126.78.244.63   |1392410692     |none                                    |550B7CD2-F0F2-591E-81E9-A76715550A3E
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |16.44.251.16    |1392410711     |none                                    |8EE2BE93-4707-5D65-BAD6-51E7D6534C18
|ai_6      |hadoop         |1_x_hadoopmaster->1_x_hadoopslave                 |173.91.93.95    |1392410716     |none                                    |53C50704-9F8E-5FE4-A3D2-F4A51BD46EA0

The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_26         |was                 |iron32      |22.17.233.19    |simhostc1                  |SUT                 |ai_1    |none       |9176C5E3-EF95-5B71-94F3-2CF9E6864D9D
|vm_27         |client_daytrader    |nano32      |19.171.57.58    |simhostd1                  |LG                  |ai_1    |none       |CB51D894-2075-552A-81EB-D2513B65EFD0
|vm_28         |db2                 |platinum64  |102.142.24.235  |simhostc1                  |SUT                 |ai_1    |none       |2A7AA310-9A19-5D98-A706-B09736FA8A97
|vm_29         |hadoopmaster        |copper32    |93.29.116.120   |simhosta1                  |SUT                 |ai_2    |none       |3D302FAA-A188-580F-AE02-33BAB4139C0E
|vm_30         |hadoopslave         |iron32      |208.156.239.154 |simhostb1                  |SUT                 |ai_2    |none       |E5FE647F-32E7-53BB-AA08-26E775039D58
|vm_31         |hadoopslave         |iron32      |152.121.15.65   |simhostb1                  |SUT                 |ai_2    |none       |23AAC528-3A8C-561C-B0B2-170E5E98D54B
|vm_32         |hadoopslave         |iron32      |136.41.209.125  |simhostb1                  |SUT                 |ai_2    |none       |786EEB12-A9B5-5842-A75B-B09B8B1924E2
|vm_33         |client_daytrader    |nano32      |100.50.216.252  |simhostd1                  |LG                  |ai_3    |none       |4B35302A-5680-5260-AC11-851E9B82A149
|vm_34         |was                 |iron32      |44.49.81.234    |simhosta1                  |SUT                 |ai_3    |none       |21511F92-16A7-5F19-B463-9A89AE084588
|vm_35         |db2                 |platinum64  |133.32.184.124  |simhostc1                  |SUT                 |ai_3    |none       |FDC5E1E5-4E0D-5D8F-A8A3-A201D10C220B
|vm_36         |client_daytrader    |nano32      |126.78.244.63   |simhostd1                  |LG                  |ai_4    |none       |FD8775F0-D9A2-5597-8B7D-E05CFD943198
|vm_37         |was                 |iron32      |34.66.51.120    |simhostc1                  |SUT                 |ai_4    |none       |9D4FF061-A308-5D28-B2DB-291FC36862DE
|vm_38         |db2                 |platinum64  |222.49.128.203  |simhostb1                  |SUT                 |ai_4    |none       |F1C2C539-ECFA-51A9-AB69-8C1315F5EB36
|vm_39         |netclient           |platinum64  |16.44.251.16    |simhostc1                  |SUT                 |ai_5    |none       |94BB8176-AAB7-5DE5-B1D3-3F46EA7BD801
|vm_40         |netserver           |iron32      |57.192.206.225  |simhosta1                  |SUT                 |ai_5    |none       |78F19F20-AE08-5609-A62D-8563B3D9B2C7
|vm_41         |hadoopmaster        |copper32    |173.91.93.95    |simhostc1                  |SUT                 |ai_6    |none       |5DA61599-5F42-5F5B-8B3D-55BEF93C90FA
|vm_42         |hadoopslave         |platinum64  |231.63.225.248  |simhostc1                  |SUT                 |ai_6    |none       |22C69AA3-46BA-59DB-A285-28EC0277F376
|vm_43         |was                 |iron32      |71.32.25.219    |simhostc1                  |SUT                 |ai_1    |none       |F7F11FEC-0B3A-5F7A-AD4A-427B6D05C459

The AI object ai_1, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (AI object key)               |value                                
|detach_parallelism                      |20                                   
|ssh_key_name                            |bogus_id_rsa                         
|sut                                     |1_x_client_daytrader->2_x_was->1_x_db2

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |3.19M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |1882415                       
|Total Connections Received                           |6200                          
|Total Commands Processed                             |1945255                       
|Number of Keys                                       |549                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.6                         
|Storage Size                                         |1642496                       
|Data Size                                            |560240                        
|Index Size                                           |457856                        
|Average Object Size                                  |531.537001898                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |18                            
|AIs                                                  |6                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |18                            
|VMs ARRIVED                                          |40                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |22                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|AI RESERVATIONS                                      |6                             
|AIs ARRIVED                                          |6                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |159                           

 ---------------- VMC-WIDE COUNTERS ----------------
|383602BF-27FC-542A-B537-6A4A58844260 (simzone_a) VM RESERVATIONS|3                             
|584A58C5-7A36-52ED-8040-423C8D700E29 (simzone_b) VM RESERVATIONS|4                             
|863DDED0-9FB6-5070-9AC8-FCF03E03B672 (simzone_c) VM RESERVATIONS|8                             
|2462767E-73BC-530A-A92C-CBED38E01A6A (simzone_d) VM RESERVATIONS|3                             

