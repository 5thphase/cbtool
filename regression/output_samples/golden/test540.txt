 status: Waiting for vm_43 (cloud-assigned uuid 18622471-8255-5174-B1D7-0E1581A00427) to start...
 status: Trying to establish network connectivity to vm_43 (cloud-assigned uuid 18622471-8255-5174-B1D7-0E1581A00427), on IP address 80.217.125.35...
 status: Bypassing the sending of a copy of the code tree to vm_43 (80.217.125.35)...
 status: Bypassing application-specific "setup" operations
AI object 1E061096-53BE-5038-9A61-797FEADA7C6F (named "ai_1") successfully resized on this experiment.
The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_1      |daytrader      |1_x_client_daytrader->2_x_was->1_x_db2            |50.121.175.89   |1381326842     |none                                    |1E061096-53BE-5038-9A61-797FEADA7C6F
|ai_2      |hadoop         |1_x_hadoopmaster->3_x_hadoopslave                 |238.26.158.198  |1381326848     |none                                    |B49B8779-31A1-51D1-B4B0-52BD1F450DF7
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |77.136.100.117  |1381326860     |none                                    |870FADCB-8EFA-51B5-A2CC-C367230522F7
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |231.86.90.94    |1381326903     |none                                    |37DC0C46-B289-52BE-B23A-838CCC59A153
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |128.1.118.199   |1381326922     |none                                    |817F3716-E289-5E74-9CFA-439D68B796FF
|ai_6      |hadoop         |1_x_hadoopmaster->1_x_hadoopslave                 |80.174.55.45    |1381326927     |none                                    |43F515CA-EDBC-5D31-8D5E-303D2C20B782

The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_26         |was                 |iron32      |180.105.157.3   |simhostb1                  |SUT                 |ai_1    |none       |9EC81ED9-6308-543B-80C3-7F82E131EB1D
|vm_27         |client_daytrader    |nano32      |50.121.175.89   |simhostd1                  |LG                  |ai_1    |none       |3AE1075C-8A50-5039-9168-037D0F318CB4
|vm_28         |db2                 |platinum64  |80.204.223.247  |simhostc1                  |SUT                 |ai_1    |none       |1DECCF72-BC5E-555E-B1FB-D3B0381D27C4
|vm_29         |hadoopmaster        |copper32    |238.26.158.198  |simhosta1                  |SUT                 |ai_2    |none       |226CEF04-0556-54B0-8C99-8D4C5F30A9FD
|vm_30         |hadoopslave         |iron32      |196.195.79.149  |simhostc1                  |SUT                 |ai_2    |none       |128A5036-99B8-56F7-9B48-C0BCE4D0B783
|vm_31         |hadoopslave         |iron32      |133.170.66.218  |simhosta1                  |SUT                 |ai_2    |none       |46BC4E5E-5E9F-5302-A9BF-72A71883E96D
|vm_32         |hadoopslave         |iron32      |219.4.106.20    |simhostb1                  |SUT                 |ai_2    |none       |46CF4B39-75DC-5FF5-BA85-EDEA6674D8CC
|vm_33         |client_daytrader    |nano32      |77.136.100.117  |simhostd1                  |LG                  |ai_3    |none       |28C3D503-AF07-539B-8058-BF5F744612EF
|vm_34         |db2                 |platinum64  |63.78.136.245   |simhosta1                  |SUT                 |ai_3    |none       |FCDF7F97-76C0-59AA-98AD-54F86FB54941
|vm_35         |was                 |iron32      |9.47.241.191    |simhosta1                  |SUT                 |ai_3    |none       |8E53C5F9-7B69-5C0D-A576-F89F5E1C4661
|vm_36         |client_daytrader    |nano32      |231.86.90.94    |simhostd1                  |LG                  |ai_4    |none       |D74FF3C7-0748-5DF4-95AA-3C5CA991145F
|vm_37         |was                 |iron32      |28.52.95.113    |simhostc1                  |SUT                 |ai_4    |none       |3DCEBD35-7918-5ECA-B5BE-754974C9551D
|vm_38         |db2                 |platinum64  |237.245.160.7   |simhosta1                  |SUT                 |ai_4    |none       |F7D8E555-DA9B-5B14-ABF6-21B93BC4BD78
|vm_39         |netserver           |iron32      |176.49.35.238   |simhostc1                  |SUT                 |ai_5    |none       |B429FD85-77F3-5CDA-BDA3-BE6BAE29F359
|vm_40         |netclient           |platinum64  |128.1.118.199   |simhosta1                  |SUT                 |ai_5    |none       |44CD170B-2DBF-5057-A049-F0158C7B2C65
|vm_41         |hadoopmaster        |copper32    |80.174.55.45    |simhostc1                  |SUT                 |ai_6    |none       |3B1EE967-AA00-5EE8-AA3A-4DA0CD1E318F
|vm_42         |hadoopslave         |platinum64  |253.169.149.179 |simhostc1                  |SUT                 |ai_6    |none       |60789D64-3139-5E17-826C-3E30091392A2
|vm_43         |was                 |iron32      |80.217.125.35   |simhostb1                  |SUT                 |ai_1    |none       |5A2494AE-E4CD-5A41-95BE-897B509B6DA6

The AI object ai_1, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (AI object key)               |value                                
|detach_parallelism                      |20                                   
|ssh_key_name                            |bogus_id_rsa                         
|sut                                     |1_x_client_daytrader->2_x_was->1_x_db2

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |3.84M                         
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |2417552                       
|Total Connections Received                           |4588                          
|Total Commands Processed                             |3947841                       
|Number of Keys                                       |549                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.5                         
|Storage Size                                         |5448777728.0                  
|Data Size                                            |4962438824.0                  
|Index Size                                           |133824768                     
|Average Object Size                                  |8101.61025917                 
|Collections                                          |40                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |18                            
|AIs                                                  |6                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |18                            
|VMs ARRIVED                                          |40                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |22                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|AI RESERVATIONS                                      |6                             
|AIs ARRIVED                                          |6                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |159                           

 ---------------- VMC-WIDE COUNTERS ----------------
|CEA17C37-820A-507F-9356-90956267A568 (simzone_a) VM RESERVATIONS|6                             
|966D86CB-90BB-553D-A06A-0FFA0818D568 (simzone_d) VM RESERVATIONS|3                             
|C41BC369-F411-5382-B95F-70C31BD20E23 (simzone_c) VM RESERVATIONS|6                             
|2FF9728E-95B3-5EE9-A0A4-61875B507FA8 (simzone_b) VM RESERVATIONS|3                             

